AmplifyPlugins/Predictions/AWSPredictionsPluginTests/Service/PredictionsTest/PredictionsServiceRekognitionTests.swift

```diff
@@ -332,7 +333,43 @@ class PredictionsServiceRekognitionTests: XCTestCase {
         }
 
         let errorReceived = expectation(description: "Error should be returned")
-        errorReceived.expectedFulfillmentCount = 2
+
+        predictionsService.detectLabels(image: url, type: .all) { event in
+            switch event {
+            case .completed(let result):
+                XCTFail("Should not produce result: \(result)")
+            case .failed(let error):
+                XCTAssertNotNil(error, "Should produce an error")
+                errorReceived.fulfill()
+            }
+        }
+
+        waitForExpectations(timeout: 1)
+    }
```

Comment:
Not sure why this is showing as new changes

Reply:
That often indicates you're based against a different branch point than you thought you were. Do you intend this to go into `main`, or did you base it on a feature branch?

Reply:
I checkout this branch from `main` and want this to go into `main`

Reply:
I do see that it's removed the expected fulfillment count as well. Double check that the change is appropriate before you merge

Reply:
This changes look ok. I think this happened because you added another test. 
You can see the change before PR after the test below. Check the line number.